src/js/widgets/recommended/index.js

```diff
@@ -26,66 +25,83 @@ function widgetLoad( {
 	} else {
 		fullUrl += `&url=${ encodeURIComponent( permalink ) }`;
 	}
-	let parentDiv = jQuery.find( `#${ widgetId }` );
+
+	let parentDiv = document.querySelector( `#${ widgetId }` );
 	if ( parentDiv.length === 0 ) {
-		parentDiv = jQuery.find( '.Parsely_Recommended_Widget' );
+		parentDiv = document.querySelector( '.Parsely_Recommended_Widget' );
 	}
 	// make sure page is not attempting to load widget twice in the same spot
-	if ( jQuery( parentDiv ).find( 'div.parsely-recommendation-widget' ).length > 0 ) {
+	if ( parentDiv.querySelectorAll( 'div.parsely-recommendation-widget' ).length > 0 ) {
 		return;
 	}
 
-	const outerDiv = jQuery( '<div>' ).addClass( 'parsely-recommendation-widget' ).appendTo( parentDiv );
+	const outerDiv = document.createElement( 'div' );
+	outerDiv.className = 'parsely-recommendation-widget';
+	parentDiv.appendChild( outerDiv );
+
 	if ( imgSrc !== 'none' ) {
-		outerDiv.addClass( 'display-thumbnail' );
+		outerDiv.classList.add( 'display-thumbnail' );
 	}
 	if ( displayDirection ) {
-		outerDiv.addClass( 'list-' + displayDirection );
+		outerDiv.classList.add( 'list-' + displayDirection );
 	}
 
-	const outerList = jQuery( '<ul>' ).addClass( 'parsely-recommended-widget' ).appendTo( outerDiv );
-	jQuery.getJSON( fullUrl, function( data ) {
-		jQuery.each( data.data, function( key, value ) {
-			const widgetEntry = jQuery( '<li>' )
-				.addClass( 'parsely-recommended-widget-entry' )
-				.attr( 'id', 'parsely-recommended-widget-item' + key );
+	const outerList = document.createElement( 'ul' );
+	outerList.className = 'parsely-recommended-widget';
+	outerDiv.appendChild( outerList );
 
-			const textDiv = jQuery( '<div>' ).addClass( 'parsely-text-wrapper' );
+	fetch( fullUrl )
```

Comment:
The [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) doesn't support IE11, but our [`defaults` for browserslist](https://github.com/Parsely/wp-parsely/blob/fcc8b363b0ea20a840f1ec7c52cd12aa1688f819/package.json#L32-L34) includes IE11 (run `npx browserslist` in the plugin directory to get the full list).

Since the plugin still supports WP 4.0, then we either need to make a call and document about the widgets not supporting IE11, or not use / provide a workaround for the Fetch API.

Reply:
If we want to provide a workaround, we could use a Polyfill: https://github.com/github/fetch

However, I'm happy to stop supporting IE11. We can clearly state it in the widget configuration.

Reply:
Internal stats suggest that IE11 requests account for around 0.1% of requests.

We should also check the behaviour of what the page looks like (is it empty? Does it have a heading but no content? I think there's an issue for this already), but in terms of using `fetch()`, ✅.

Reply:
Actually, there is an open issue about dropping IE 11 in the upcoming release: https://github.com/Parsely/wp-parsely/issues/325

Reply:
> We should also check the behaviour of what the page looks like (is it empty? Does it have a heading but no content? I think there's an issue for this already)

https://github.com/Parsely/wp-parsely/pull/193